src/Controller/UI/DatasetReviewController.php

```diff
@@ -532,10 +507,18 @@ public function postAction(Request $request, int $id = null, MessageBusInterface
                     }
                     $datasetSubmission = $datasetSubmission->getDataset()->getDatasetSubmission();
                     $datasetSubmission->setFileset($newFileset);
-                    $this->entityHandler->update($datasetSubmission);
+                    $entityManager->persist($newFileset);
                 }
             }
 
+            $entityManager->flush();
+
+            // update MDAPP logs after action is executed.
```

Comment:
I don't think mdapp still exists. But why is this now added? (The event dispatch)

Reply:
I moved it, and i think we still have the

Reply:
Yes, it still exists as a place to hold certain logs. This is indeed old, but I think still used, primarly just for the log content.